dnsprovider/pkg/dnsprovider/providers/scaleway/dns.go

```diff
@@ -353,75 +353,35 @@ func (r *resourceRecordChangeset) Apply(ctx context.Context) error {
 		return err
 	}
 
-	if len(r.additions) > 0 {
-		recordsToAdd := []*domain.Record(nil)
-		for _, rrset := range r.additions {
-			recordName := strings.TrimSuffix(rrset.Name(), ".")
-			recordName = strings.TrimSuffix(recordName, "."+r.zone.Name())
-			for _, rrdata := range rrset.Rrdatas() {
-				recordsToAdd = append(recordsToAdd, &domain.Record{
-					Name: recordName,
-					Data: rrdata,
-					TTL:  uint32(rrset.Ttl()),
-					Type: domain.RecordType(rrset.Type()),
-				})
-			}
-			klog.V(8).Infof("adding new DNS record %q to zone %q", recordName, r.zone.name)
-			updateRecordsRequest = append(updateRecordsRequest, &domain.RecordChange{
-				Add: &domain.RecordChangeAdd{
-					Records: recordsToAdd,
-				},
-			})
-		}
-	}
-
+	// Scaleway's Domain API doesn't allow edits to the same record if one request, so we have to check for duplicates
```

Comment:
```suggestion
	// Scaleway's Domain API doesn't allow edits to the same record in one request, so we have to check for duplicates
```
I'm not sure I'm understanding this sentence correctly.